source/examples/generated/code/start/AddSyncToApp.snippet.authenticate-user.swift

```diff
@@ -0,0 +1,7 @@
+func login() async throws -> User {
+    // Authenticate with the instance of the app that points
```

Comment:
```suggestion
    // Authenticate with the instance of the App that points
```

should this be uppercase, as in 'App Services App'? not sure from context, just wanted to check.

Reply:
In this case, we're referring to lowercase app which is the variable name for our App instance. But good catch! I do tend to conflate the two.